subiquity/server/controllers/zdev.py

```diff
@@ -593,10 +596,43 @@
 id="0.0.c0fe" type="generic-ccw" on="no" exists="yes" pers="no" auto="no" failed="yes" names=""'''  # noqa: E501
 
 
+ZdevAiItem = TypedDict(
```

Comment:
For most things like this we been doing attrs, what is the motivation here for TypedDict instead?

Reply:
got it.  I still lean towards the attrs implementation to stay consistent with other places in Subiquity.  Can we try it in a new commit and see if it reads differently?

Reply:
Chris and I updated the code to use attrs as well (in the last commit)